simpeg/maps/__init__.py

```diff
@@ -0,0 +1,35 @@
+from ._base import (
+    ComboMap,
+    IdentityMap,
+    LinearMap,
+    Projection,
+    SelfConsistentEffectiveMedium,
```

Comment:
This one is mesh agnostic and is pretty specialized, so I don't think it should be in the `base`